src/panels/lovelace/cards/hui-entity-button-card.ts

```diff
@@ -210,14 +211,38 @@ class HuiEntityButtonCard extends LitElement implements LovelaceCard {
         color: var(--paper-item-icon-color, #44739e);
       }
 
-      ha-icon[data-domain="light"][data-state="on"],
-      ha-icon[data-domain="switch"][data-state="on"],
+      ha-icon[data-domain="alarm_control_panel"][data-state="disarmed"],
```

Comment:
Rip out all this CSS from both places and please extract it into a constant.

Reply:
so a single `active/heat/cool/unavailable` attribute?